wlct/cogs/ladders.py

```diff
@@ -3,16 +3,114 @@
 from wlct.tournaments import Tournament, TournamentTeam, TournamentPlayer, MonthlyTemplateRotation, get_games_finished_for_team_since, find_tournament_by_id, get_team_data_no_clan, RealTimeLadder, get_real_time_ladder, TournamentGame
 from wlct.logging import log_bot_msg, log_exception
 from discord.ext import commands, tasks
-from wlct.cogs.common import is_admin
+from wlct.cogs.common import is_admin, is_tournament_creator
 from django.utils import timezone
 from traceback import print_exc
 
+
+
 class Ladders(commands.Cog, name="ladders"):
     ''' Actually sends the help command '''
 
     def __init__(self, bot):
         self.bot = bot
 
+    async def ladder_command(self, ctx, ladder, cmd, option):
+        invalid_cmd_text = "You've entered an invalid command. bb!rtl no longer takes in a ladder id. Please see ``bb!help rtl`` for an updated list of commands."
+        emb = discord.Embed(color=self.bot.embed_color)
+        emb.set_author(icon_url=ctx.message.author.avatar_url, name=ctx.message.author)
+        emb.set_footer(text="Bot created and maintained by -B#0292")
+        do_embed = False
+        discord_id = ctx.message.author.id
+        teams = ladder.get_active_team_count()
+        if cmd == "-p":
+            # display current players in the ladder
+            retStr = ladder.get_current_joined()
+        elif cmd == "-j":
+            retStr = ladder.join_ladder(discord_id, False)
+            current_joined = ladder.get_current_joined()
+            retStr += "\n\n" + current_joined + "\n"
+            log_bot_msg("User {} has joined the RTL. New team count: {}".format(ctx.message.author.name, teams))
```

Comment:
Not really important now, but for logging might help to specify which ladder specifically. (Same comment for bot logs below as well)